<ui version="4.0" >
 <class>QgsInterpolationDialogBase</class>
 <widget class="QDialog" name="QgsInterpolationDialogBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>538</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="mInputGroupBox" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Input</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="mInputVectorLayerLabel" >
          <property name="text" >
           <string>Input vector layer:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="mInputLayerComboBox" />
        </item>
       </layout>
      </item>
      <item row="1" column="0" >
       <widget class="QCheckBox" name="mUseZCoordCheckBox" >
        <property name="text" >
         <string>Use z-Coordinate for interpolation</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="mInterpolationAttributeLabel" >
          <property name="text" >
           <string>Interpolation attribute: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="mInterpolationAttributeComboBox" />
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="mOutputGroupBox" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Output</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" colspan="2" >
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="mInterpolationLabel" >
          <property name="text" >
           <string>Interpolation method:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="mInterpolationMethodComboBox" />
        </item>
       </layout>
      </item>
      <item row="1" column="0" >
       <widget class="QPushButton" name="mConfigureInterpolationButton" >
        <property name="text" >
         <string>Configure interpolation method...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>51</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0" colspan="2" >
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="mNumberOfColumnsLabel" >
          <property name="text" >
           <string>Number of columns:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="mNumberOfColumnsSpinBox" >
          <property name="maximum" >
           <number>10000000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0" colspan="2" >
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="mNumberOfRowsLabel" >
          <property name="text" >
           <string>Number of rows:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="mNumberOfRowsSpinBox" >
          <property name="maximum" >
           <number>10000000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0" colspan="2" >
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="mOutputFileLabel" >
          <property name="text" >
           <string>Output File: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="mOutputFileLineEdit" />
        </item>
        <item>
         <widget class="QPushButton" name="mOutputFileButton" >
          <property name="text" >
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QgsInterpolationDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QgsInterpolationDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
